test/near_provider.test.js

```diff
@@ -136,36 +136,28 @@ describe('\n---- PROVIDER ----', () => {
     describe('\n---- BASIC QUERIES ----', () => {
         describe('getProtocolVersion | eth_protocolVersion', () => {
             test('returns protocol version', withWeb3(async (web) => {
-                try {
-                    // Get current version
-                    const { version: { version } } = await testNearProvider.status();
-                    const getVersion = await web.eth.getProtocolVersion();
+                // Get current version
+                const { version: { version } } = await testNearProvider.status();
+                const getVersion = await web.eth.getProtocolVersion();
 
-                    expect(typeof version).toStrictEqual('string');
-                    expect(web3.utils.toHex(version)).toStrictEqual(getVersion);
-                } catch (e) {
```

Comment:
unnecessary??

Reply:
Probably. I suppose we only need to catch errors if we are expecting an error since Jest will log it anyway.